packages/esm-bed-management-app/src/bed-administration/form/add-edit-bed.resource.ts

```diff
@@ -0,0 +1,117 @@
+import { useMemo } from 'react';
+import useSWR from 'swr';
+import { type FetchResponse, openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+
+interface BedForm {
+  bedNumber: string;
+  bedType: string;
+  row: number;
+  column: number;
+  status: string;
+  locationUuid: string;
+  uuid?: string;
+}
+
+interface BedTag {
+  id?: string;
+  name: string;
+  uuid?: string;
+}
+export interface BedPostPayload {
+  uuid?: string;
+  bedNumber: string;
+  bedType: string;
+  row: number;
+  column: number;
+  status: string;
+  locationUuid: string;
+  bedTag?: BedTag[];
+}
```

Comment:
```suggestion
```

Shouldn't we extend the similarly named interface in `types.ts` instead of creating a duplicate?